pkgs/tools/games/slipstream/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, fetchFromGitHub, stdenv, makeWrapper, buildMaven, maven, jdk8 }:
+let
+  jdk = jdk8;
+  mavenWithJdk = maven.override { inherit jdk; };
+in
+mavenWithJdk.buildMavenPackage rec {
+  pname = "slipstream";
+  version = "1.9.1";
+
+  src = fetchFromGitHub {
+    owner = "Vhati";
+    repo = "Slipstream-Mod-Manager";
+    rev = "v${version}";
+    hash = "sha256-F+o94Oh9qxVdfgwdmyOv+WZl1BjQuzhQWaVrAgScgIU=";
+  };
+
+  mvnHash = "sha256-oDtUitsfZPiDtyfzzw1yMNBCKyP6rHczKZT/SPPJYGE=";
+
+  nativeBuildInputs = [ mavenWithJdk makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/java
+    install -Dm644 target/ftl-mod-manager-${version}.jar $out/share/java
+    install -Dm644 target/modman.jar $out/share/java
+
+    makeWrapper ${wrapper} $out/bin/${pname} \
+      --suffix PATH : ${lib.makeBinPath [ jdk ]} \
+      --set jar_file "$out/share/java/modman.jar"
+
+    runHook postInstall
+  '';
+
+  wrapper = ./wrapper.sh;
```

Comment:
Is this wrapper really necessary ? It looks weird.

Reply:
I'm not sure of an alternative but I'm certainly open to it. I suppose it might be possible with `makeWrapper`?

Reply:
I do not feel comfortable with this as well, I would find it proper if embedded in the derivation:
```nix
{
  let
    wrapper = ./wrapper.sh
  in mkDerivation {
    installPhase = ''
      makeWrapper ${wrapper}
    '';
  }
}
```